indico/modules/events/registration/templates/management/regform_invitations.html

```diff
@@ -21,6 +21,7 @@
                     </div>
                 </div>
                 <div class="toolbar">
+                    {{ template_hook('invite-options', event=event, regform=regform) }}
```

Comment:
```suggestion
                    {{ template_hook('registration-invite-options', event=event, regform=regform) }}
```

Just to be clearer of the scope.